packages/core-data/src/selectors.js

```diff
@@ -95,27 +95,62 @@ export const getUserQueryResults = createSelector(
 );
 
 /**
- * Returns whether the entities for the give kind are loaded.
+ * Returns the loaded entities for the given kind.
  *
+ * @deprecated since ??. Use getEntitiesConfig instead
```

Comment:
Is the idea to deprecate it starting from WP 6.0? Let's be explicit because it's very confusing as it can be read as:
- npm version
- Gutenberg version
- WordPress version

Very much depends on the context you consume it 😅 

```suggestion
 * @deprecated since WordPress 6.0. Use getEntitiesConfig instead.
```

Reply:
I left it as a placeholder since I wasn't sure what to say there :D thanks!